cfgmgr/teammgrd.cpp

```diff
@@ -55,6 +67,12 @@ int main(int argc, char **argv)
 
         while (true)
         {
+            if(received_sigterm)
+            {
+                teammgr.cleanTeamProcesses(SIGTERM);
+                received_sigterm = false;
```

Comment:
should we stop this process too?

Reply:
Since we need to have Signal Handlers in both the teammgrd and teamsyncd and both have resources to be cleaned up which are "interdependent", I found it is right to let the process continue and get killed by SIGKILL later. If we add explicit process exits, since they are exiting on their on pace, on testing I found some of the PortChannel interfaces remaining in the kernel and not getting cleaned.